babel.config.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-  presets: [['@shopify/babel-preset/node', {modules: 'commonjs'}]],
+  presets: [['@shopify/babel-preset/common', {modules: 'commonjs'}]],
```

Comment:
You're no longer specifying a target for babel to compile to (as previously the node preset defined it) so it will now compile everything down to es5, which is way overkill.

At a guess this is why you had to add regenerator-runtime deps - as tests failed because the transform down to es5 transpiled async code to use generators, which depends on regenerator-runtime.

Targeting `current node` should stop that transform from occurring - the node versions in dev.yml / CI support `async` nativly.